.github/workflows/performance-checks.yml

```diff
@@ -0,0 +1,36 @@
+name: Performance-Checks
+
+on:
+  workflow_call:
+
+jobs:
+  performance-checks:
+    name: Run Performance-Checks (Python-${{ matrix.python-version }}, Exasol-${{ matrix.exasol-version}})
+    runs-on:  ubuntu-24.04
+    timeout-minutes: 30
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.11"]
```

Comment:
double-check: we only want for 1 python version & db?

Reply:
Yeah, I think for the moment, one of each is enough. However, we usually work with the even numbered python versions. So, python 3.10 or 3.12. We are switching to 3.12 soon in the UDFs so probably this